config/jest/jest.config.js

```diff
@@ -1,17 +1,21 @@
-const { join } = require("path");
+import { dirname, join } from "path";
+import { fileURLToPath } from "url";
 
-module.exports = {
-  rootDir: join(__dirname, "..", ".."),
+const filePath = fileURLToPath(import.meta.url);
```

Comment:
Cannot use `__dirname` or `require` since there is `"type": "module"` in the package.json.